templates/helm/values.yaml.tpl

```diff
@@ -15,6 +15,10 @@ deployment:
   annotations: {}
   labels: {}
   containerPort: 8080
+  # Number of Deployment replicas
+  # This determines how many instances of the controller will be running. It's recommended
+  # to have more than one replica (>1) only when leader election is enabled.
```

Comment:
Reverse the wording here and say "It's recommended to enable leader election if you need to increase the number of replicas > 1"